### PR TITLE
editing functionallity added

### PR DESCRIPTION
**Pull request by Dev-Pancea**

### CODE REVIEW

**Review by SkylarSLiang — COMMENTED** [review]
LGTM!

**Review by Aditya-wsib — COMMENTED** [review]
added singular limit for edit feature, awaiting approval from other devs

**Comment by bilal-wsib** [comment]
> added singular limit for edit feature, awaiting approval from other devs

LGTM we can merge this into master, and I can do the storage stuff in a seperate branch and pr

File: testing/src/components/Container.js [hunk 1]
```diff
@@ -30,24 +75,30 @@ export const Container = () => {
...
+          </button>
+
+          {items.map((item, index) => {
+            return (item.edit && (isEdit === 1)) ? (
+              <ItemEdit
+                updateVal={updateVal}
+                onSaveEdit={onSaveEdit}
+                cancelEditItem={cancelEditItem}
+                item={item}
+                index={index}
```
Comment (bilal-wsib) [inline testing/src/components/Container.js]: you wanna do `key={index}` in this situation to stop the warning

Comment (SkylarSLiang) [reply testing/src/components/Container.js]: gotcha!

**Review by Aditya-wsib — APPROVED** [review]

**Pull request merged**